README.md

```diff
@@ -10,218 +10,326 @@
 ```
-On Ubuntu and Raspberry Pi OS you can get the libraries by running
+_Windows_
+```bat
+choco install gstreamer --version=1.22.8 gstreamer-devel --version=1.22.8
 ```
-$ sudo apt-get install libssl-dev libcurl4-openssl-dev liblog4cplus-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-plugins-base-apps gstreamer1.0-plugins-bad gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly gstreamer1.0-tools
+#### Verify GStreamer Installation
+Run the following command to display the GStreamer version to confirm the installation was successful:
+```bash
+gst-launch-1.0 --gst-version
 ```
-### Setup desired log level:
-Set up the desired log level. The log levels currently available with `log4cplus` are:
-1. `TRACE`
-2. `DEBUG` 
-3. `INFO`   
-4. `WARN`   
-5. `ERROR`   
-6. `FATAL`   
 
-To set a log level, update the log level value [here](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/blob/master/kvs_log_configuration#L1)
+<br>
 
-Note: The default log level is `DEBUG`
-
-#### Cross-Compilation
-If you wish to cross-compile `CC` and `CXX` are respected when building the library and all its dependencies. See our [ci.yml](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/blob/develop/.github/workflows/ci.yml) for an example of this. Every commit is cross compiled to ensure that it continues to work.
-Please note that GStreamer is not cross-compiled as a part of the cross-compilation of the KVS-SDK, customers will have to cross-compile it separately.
+### Download
+```bash
+git clone https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp.git --single-branch -b master kvs-producer-cpp-sdk
+```
 
+<br>
 
-#### CMake Arguments
-You can pass the following options to `cmake ..`.
+### Build
+#### Prepare a build directory in the newly checked out repository:
+```bash
+mkdir -p kvs-producer-cpp-sdk/build
+cd kvs-producer-cpp-sdk/build
+```
+#### Configure the build:
 
-* `-DBUILD_GSTREAMER_PLUGIN` -- Build kvssink GStreamer plugin
-* `-DBUILD_JNI` -- Build C++ wrapper for JNI to expose the functionality to Java/Android
-* `-DBUILD_DEPENDENCIES` -- Build depending libraries from source
-* `-DBUILD_TEST=TRUE` -- Build unit/integration tests, may be useful for confirm support for your device. `./tst/producerTest`
-* `-DCODE_COVERAGE` --  Enable coverage reporting
-* `-DCOMPILER_WARNINGS` -- Enable all compiler warnings
-* `-DADDRESS_SANITIZER` -- Build with AddressSanitizer
-* `-DMEMORY_SANITIZER` --  Build with MemorySanitizer
-* `-DTHREAD_SANITIZER` -- Build with ThreadSanitizer
-* `-DUNDEFINED_BEHAVIOR_SANITIZER` Build with UndefinedBehaviorSanitizer
-* `-DALIGNED_MEMORY_MODEL` Build for aligned memory model only devices. Default is OFF.
-* `-DBUILD_LOG4CPLUS_HOST` Specify host-name for log4cplus for cross-compilation. Default is OFF.
+_Mac and Linux_
+```bash
+cmake -DBUILD_GSTREAMER_PLUGIN=TRUE ..
```

Comment:
Just for consistency, let's use ON instead of TRUE. There's a few other places to adjust as well.